src/demo-app/demo-app.html

```diff
@@ -1,7 +1,57 @@
 <p>
   material2 Works!
-
   <button md-button>HELLO</button>
   <button md-raised-button class="md-primary">HELLO</button>
   <button md-fab class="md-accent">HI</button>
 </p>
+<div class="card-container">
+
+  <md-card>
+    <md-card-title-group>
+      <md-card-title>Card with title</md-card-title>
+      <md-card-subtitle>Subtitle</md-card-subtitle>
+      <img md-card-md-image src="https://material.angularjs.org/latest/img/washedout.png">
```

Comment:
You know, I've never liked this image we've been using for a card example... let me see if I can find something better.